api/modules/publishedProjects/model.js

```diff
@@ -13,6 +13,34 @@ var instanceProps = {
   publishedFiles: function() {
     return this.hasMany(require('../publishedFiles/model'));
   },
+  format: function(model) {
```

Comment:
code duplication; it makes more sense to refactor this to its own file and using something like
```
format: require('utils/datehandling/format'),
pares: require('utils/datehandling/parse'),
...
```